src/python/WMCore/Services/SiteDB/SiteDBAPI.py

```diff
@@ -33,7 +33,7 @@ class SiteDBAPI(Service):
 
     def __init__(self, config={}, logger=None):
         config = dict(config)
-        if os.getenv("WMAGENT_USE_CRIC", False):
+        if os.getenv("WMAGENT_USE_CRIC", False) or os.getenv("WMCORE_USE_CRIC", False):
             # just to make sure we don't use SiteDB anywhere when CRIC flag is true
             config.setdefault('endpoint', "https://BLAH.cern.ch/sitedb/data/prod/")
```

Comment:
So you want to crash the component if SiteDB is used, right? Can we just raise an exception?

Reply:
It could have been done. I was afraid there could be a place in the code we get an instance of SiteDB without actually using it, thus I didn't want to have an exception for that case. However, it would probably have been a better idea.
If you want I can create another patch for this, but I think it's really no longer used, otherwise I'd have hit this problem in my tests.